app/views/taxons/edit.html.erb

```diff
@@ -10,4 +10,9 @@
       input_html: { class: :select2, multiple: true, include_blank: true } %>
 
   <%= f.submit I18n.t('views.taxons.edit_button'), class: "btn btn-lg btn-success" %>
+
+  <%= link_to 'Delete taxon', taxon_path(taxon.content_id),
+    method: :delete,
+    data: { confirm: I18n.t('messages.views.confirm') },
+    class: 'btn btn-lg btn-danger' %>
```

Comment:
I've removed it altogether. Adding it to the header causes weird spacing issues, but more crucially I think Mark's point in adding a delete link to the show page was to allow the user to see the content they would be 'unlinking' from a taxon before hitting the button.

For an action the user is unlikely to do very often, I'm not sure it makes sense to include two different ways to delete a taxon. I think we should leave this for now and only add it back in if this raises confusion or seems missing to users.